libs/ha/ha_common_libs_k8s.py

```diff
@@ -888,7 +888,7 @@ def restore_pod(pod_obj, restore_method, restore_params: dict = None):
 
     def event_s3_operation(self, event, setup_s3bench=True, log_prefix=None, s3userinfo=None,
                            skipread=False, skipwrite=False, skipcleanup=False, nsamples=10,
-                           nclients=10, output=None, event_set_clr=None):
+                           nclients=10, output=None, event_set_clr=None, httpclientimeout=300000):
```

Comment:
param name can be only 'timeout'. This name is quite big.

Reply:
> param name can be only 'timeout'. This name is quite big.

there are many type of timeout in s3bench command. Hence to keep it specific used as it is mentioned in s3bench docs.